tests/scipy_optimize_test.py

```diff
@@ -0,0 +1,259 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from absl.testing import absltest
+
+import jax
+from jax import numpy as jnp
+from jax import test_util as jtu
+import jax.scipy.sparse.linalg
+from jax.config import config
+from jax.scipy.optimize.bfgs_minimize import BFGSResults, bfgs_minimize
+from jax.scipy.optimize.line_search import LineSearchResults, line_search
+
+config.parse_flags_with_absl()
+
+from jax.config import config
+
+config.parse_flags_with_absl()
```

Comment:
please delete this redundant call to parse_flags_with_absl()

(it looks like you copied it from https://github.com/google/jax/pull/3128 but it isn't necessary)